simpledotorg/bp-passport
# API - login and loading basic patient data from api

To test you'll need to:

Run on a real device.
Go here: https://www.qr-code-generator.com/ and create a QR code with this id that will authenticate: '86d89f24-fc11-4829-aa4e-5daee20a370a'

When you get the stage 2 - the verify number screen - 000000 will work as the OTP to complete the authentication.

When you first enter the home screen you'll see that the username takes a few seconds to load. This is when the home screen should be displaying loading ui.
